packages/documentation/pages/usage/components/form-fields.vue

```diff
@@ -448,6 +448,15 @@
 								label="option slot"
 								type="switch"
 							/>
+							<KtFieldToggle
+								v-if="
+									componentDefinition.additionalProps.includes('clearOnSelect')
+								"
+								formKey="clearOnSelect"
+								isOptional
+								label="clear on select"
```

Comment:
Should be
```suggestion
								label="clearOnSelect"
```
as this serves as the documentation for how this prop is called as well